client/assets/less/pages/places.less

```diff
@@ -249,7 +240,89 @@
     width: 100%;
     @media (max-width: 767px) {
       .leaflet-control-zoom {
-        display: none;
+        top: 50px;
+      }
+    }
+  }
+}
+
+// Icons for popular places and tracks in location
+.popular-places-wrapper, .popular-tracks-wrapper {
+  display: none;
+  position: absolute;
+  z-index: 2;
+  top: 0;
+  right: 0;
+  padding-top: 115px;
+  background-color: #f8f8f8;
+  transition: all linear 0.5s;
+  width: 25%;
+  height: 100%;
+  &.ng-hide {
+    opacity: 0;
+  }
+  a:first-child {
```

Comment:
This close button is under images on scroll so maybe need z-index.
Also this is close button so you can add some class for it - it will be more informative.